### PR TITLE
Script to convert WDL to DOT

### PR DESCRIPTION
**Pull request by meganshand**
I think at the moment I want this to work as a standalone script so I left the import as written and put it in the scripts directory. I can also include an example and readme if you think that's appropriate here, just let me know. If there's a way to get this tool more integrated so that it would be easier to use that would be great too.

Currently this prints the tree from beginning the pipeline at the top of the tree to the end of the pipeline at the bottom. It places unlabeled boxes around scatter blocks.

### CODE REVIEW

**Comment by scottfrazer** [comment]
I've merged your other PR and uploaded version 1.0.18 to PyPI

**Comment by scottfrazer** [comment]
This could use a little bit of documentation in the README file, but besides that it looks pretty good to me

File: README.md [hunk 1]
```diff
@@ -350,3 +351,18 @@ $ wdl parse examples/ex2.wdl
         (RawCommand:
 ...
 ```
+
+# Converting To DOT
+
+A wdl file can be converted to the dot format in order to be able to visualize the pipeline as a graph. For example:
+
+```
+$ python scripts/wdl2dot.py -i hello.wdl -o hello.dot
```
Comment (scottfrazer) [inline README.md]: Can you rename `scripts/wdl2dot.py` to just `scripts/wdl2dot`, then do a `chmod +x` on it?

Then, in the `setup.py` file, add it to the scripts section:

```
scripts={
   'scripts/wdl',
   'scripts/wdl2dot'
}
```

This way, this line of the documentation can just be:

```
wdl2dot -i hello.wdl -o hello.dot
```

Comment (scottfrazer) [inline README.md]: This will also have the added benefit of making `wdl2dot` available to people who install PyWDL via `pip install pywdl`

**Comment by meganshand** [comment]
I just double checked the output and found something wrong. I'll try to figure it out.

**Comment by meganshand** [comment]
Oh never mind, I forgot to rebase. It should work correctly now.

**Pull request merged**